### ISSUE TITLE
Mouse cursor hides while dragging texture to sprite

### ISSUE THREAD

**Issue by TaofeekS** [godotengine/godot#47608]
**Godot version:**
Godot 3.2.3

**OS/device including version:**
Windows 10 GLES3 

**Issue description:**
My mouse pointer just hides and never show on screen again sometimes while i am dragging a texture to my sprite when using animation player to animate it. The pointer will never show up again unless I restart Godot

**Steps to reproduce:**
Open a godot scene
add a sprite and an animation player to the scene.
Try dragging textures into the sprite and animating it with the animation player

Note: This is not a regular occurrences 

**Minimal reproduction project:**

**Comment by Calinou** [comment]
This is likely related to either mouse cursor changing or mouse mode being changed to `MOUSE_MODE_CAPTURED` while dragging a SpinBox or EditorSpinSlider.

**Comment by TaofeekS** [comment]
Yes i suspect `MOUSE_MODE_CAPTURED` but how can i fix it. it just toggle on by itself and it sometimes annoying to just restart the app again

**Comment by Calinou** [comment]
> Yes i suspect `MOUSE_MODE_CAPTURED` but how can i fix it. it just toggle on by itself and it sometimes annoying to just restart the app again

As a workaround, you can click the `[empty]` text (or the arrow besides it), choose **Load** and select the texture there.
